src/models/event.ts

```diff
@@ -404,8 +404,12 @@ export class MatrixEvent extends TypedEventEmitter<MatrixEventEmittedEvents, Mat
         });
 
         this.txnId = event.txn_id;
+        // The localTimestamp is calculated using the age.
+        // Some homeserver don't provide the unsigned field for state events.
```

Comment:
```suggestion
        // Some events lack an `age` property, either because they are EDUs such as typing events,
        // or due to server-side bugs such as https://github.com/matrix-org/synapse/issues/8429.
```